index.html

```diff
@@ -1,19 +1,275 @@
+          <button>Learn More</button>
+        </div>
+      </main>
+    </div>
+
+    <h2>Everyone needs a little Karma.</h2>
+
+    <section class="icons__section">
+      <div class="icon">
+        <img
+          src="img/icon-devices.svg"
+          alt="three devices"
+          class="icon__logo"
+        />
+        <p>Internet for all devices</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-coffee.svg" alt="coffee cup" class="icon__logo" />
+        <p>Boost your productivity</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-refill.svg" alt="charger icon" class="icon__logo" />
+        <p>Pay as You Go</p>
+      </div>
+    </section>
+    
+    <section class="features">
+      <div class="rectangle">
+        <p class="quote">
+          Wherever I am, I just don't worry about my connection anymore!
+        </p>
+        <button href="#" class="button_color">Get Karma today</button>
+      </div>
+    </section>
+
+    <footer>
+      <h3>Join us on</h3>
+      <a href="#"><img src="./img/twitter-icon.svg" alt="Twitter Logo" /></a>
+      <a href="#"><img src="./img/facebook-icon.svg" alt="Facebook Logo" /></a>
+      <a href="#"
+        ><img src="./img/instagram-icon.svg" alt="Instagram Logo"
+      /></a>
+      <p>© Karma Mobility Inc</p>
+    </footer>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+  </body>
+</html>
+
+
+<!-- Store page markup -->
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma logo" class="header__logo" />
+      <div>
+        <nav>
+          <ul class="navigation__list">
+            <li>
+              <a class="navigation__link__karma" href="index.html"
+                >Meet Karma</a
+              >
+            </li>
+            <li>
+              <a href="#">How It Works</a>
+            </li>
+            <li>
+              <a class="navigation_link" href="#">Store</a>
+            </li>
+            <li>
+              <a href="#">Blog</a>
+            </li>
+            <li>
+              <a href="#">Help</a>
+            </li>
+            <li>
+              <a href="#">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <store_main class="order_section">
```

Comment:
You have a tag here called "store_main" - I don't think this will be valid HTML. I believe you will need to stick to the valid HTML tags. If you separate the store into it's own HTML file, you could maybe make this the "main" section.

Reply:
I changed that, hope is ok now.